contents/handbook/company/management.md

```diff
@@ -40,11 +38,12 @@ A manager at PostHog is _not_ responsible for:
 At PostHog, we exclusively hire people that are the best in their field.
 That means managers won't need to spend time telling their direct reports what to do.
 
-However, for those people to make the best decisions, they need context. That context can be:
-- what a customer said was or wasn't important to them
-- what the metrics are saying needs to be improved
-- what another team in the organisation is working on
-- what the overall goals are for PostHog
+However, for those people to make the best decisions, they need context. The things a manager can do to set context include:
+- creating a roadmap that the team can work towards
+- helping someone figure out who else to talk to within PostHog
+- creating metrics that the team should improve
```

Comment:
Subtle change - but we might not always be able measure success through metrics.

```suggestion
- enabling the team to measure their impact
```